src/main/java/com/facebook/openwifirrm/optimizers/tpc/MeasurementBasedApApTPC.java

```diff
@@ -370,40 +340,76 @@ protected void buildTxPowerMapForChannel(
 				);
 				continue;
 			}
-			JsonArray radioStatuses =
-				allStatuses.get(serialNumber).getAsJsonArray();
-			Optional<Integer> possibleCurrentTxPower = getCurrentTxPower(
-				radioStatuses,
-				band
-			);
-			if (possibleCurrentTxPower.isEmpty()) {
-				// this AP is not on the band of interest
-				continue;
-			}
-			int currentTxPower = possibleCurrentTxPower.get();
-			String bssid = state.interfaces[0].ssids[0].bssid;
-			List<Integer> rssiValues = bssidToRssiValues.get(bssid);
-			logger.debug("Device <{}> : BSSID <{}>", serialNumber, bssid);
-			for (int rssi : rssiValues) {
-				logger.debug("  Neighbor received RSSI: {}", rssi);
+			State deviceState =
```

Comment:
These two lines can fit in one line

Reply:
oops forgot to run spotless I think

Reply:
never mind, i did run it and didn't do anything. will change